airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -25,8 +26,9 @@
 from airflow_prometheus_exporter.xcom_config import load_xcom_config
 
 CANARY_DAG = "canary_dag"
-RETENTION_TIME = os.environ.get("PROMETHEUS_METRICS_DAYS", 14)
+RETENTION_TIME = os.environ.get("PROMETHEUS_METRICS_DAYS", 21)
```

Comment:
Need to expand the retention time to 21 days. This is to handle the case like Figo. Figo has weekly interval. The SLA interval will have to be set to 2w. Need to grab 3 weeks of data to find out the latest previous run outside of sla window.